packages/shared/src/components/MainFeedLayout.tsx

```diff
@@ -272,6 +274,25 @@ export default function MainFeedLayout({
       return null;
     }
 
+    if (router?.pathname === '/' && isCustomDefaultFeed) {
+      return {
+        feedName: SharedFeedPage.Custom,
+        feedQueryKey: generateQueryKey(SharedFeedPage.Custom, user),
+        query: CUSTOM_FEED_QUERY,
+        variables: {
+          feedId: user.defaultFeedId,
+          feedName: SharedFeedPage.Custom,
+        },
+        emptyScreen: propsByFeed[feedName].emptyScreen || <FeedEmptyScreen />,
```

Comment:
@capJavert Can you also log that this needs a custom feedEmptyScreen, the current one will open the main feed modal

Reply:
yeah, we probably should, kinda forgot about it in the RFC, I focused on the fact that whatever your default feed is that one can be opened in the extension

Reply:
>Can you also log that this needs a custom feedEmptyScreen, the current one will open the main feed modal

I added empty screen in Feed.tsx on epic branch.